frontend/src/scenes/insights/views/InsightsTable/columns/SeriesColumn.tsx

```diff
@@ -28,7 +28,7 @@ export function SeriesColumnItem({
     const showCountedByTag = !!indexedResults.find(({ action }) => action?.math && action.math !== 'total')
 
     return (
-        <div className="series-name-wrapper-col">
+        <div className="series-name-wrapper-col space-x-1">
```

Comment:
Only visual changes are now caused by this which adds a small gap so the button hover state does not touch the text